File: angular/src/app/shared/services/student/models/create-update-student-dto.ts
import { EntityDto } from '@abp/ng.core';

export class CreateUpdateStudentDto extends EntityDto<string>{

  name: string;
  birthday: Date;
  email: string;
  phoneNumber: number;
  address: string;
  faculty: string;
  branch: string;
  courseYear: string;
  studentCode: string;

  constructor(initial: Partial<CreateUpdateStudentDto> = {}) {
    super(initial);
  }
}

File: aspnet-core/src/doan.ProjectManagement.Application.Contracts/Students/Dto/CreateUpdateStudentDto.cs
using System;
using Volo.Abp.Application.Dtos;

namespace doan.ProjectManagement.Students.Dto
{
    public class CreateUpdateStudentDto : EntityDto<Guid?>
    {
        public string Name { get; set; }
        public DateTime? Birthday { get; set; }
        public string PhoneNumber { get; set; }
        public string Email { get; set; }
        public string Address { get; set; }
        public string Faculty { get; set; }
        public string Branch { get; set; }
        public string CourseYear { get; set; }
        public string StudentCode { get; set; }
    }
}
